### PR TITLE
[RA-2] Ubuntu Docs

### PR DESCRIPTION
**Pull request by mlavin**
Adding some additional notes on getting the project running on Ubuntu.

Note: This merges these changes into the staging branch. Going forward, I think we plan on using `staging` and `master` as the two primary long lived branches.

### CODE REVIEW

File: DETAILED_INSTRUCTIONS.md [hunk 1]
```diff
@@ -66,6 +66,59 @@ Since you will someday be working on many python projects and need to keep the p
...
+
+```sh
+$ sudo pip install virtualenv virtualenvwrapper
+```
+
+This will be used later to create an isolated Python environment for the development of the project.
+
+## Search, Caching, and related packages
+
+This project uses Postrgres, Memecached, Redis, and ElasticSearch for DB storage, caching, background queuing, and search respectively. With the exception of ElasticSearch, you can use the OS provided versions of these services installable via `apt`:
```
Comment (daaray) [inline DETAILED_INSTRUCTIONS.md]: spelling: Postrgres, Memecached

File: DETAILED_INSTRUCTIONS.md [hunk 2]
```diff
@@ -66,6 +66,59 @@ Since you will someday be working on many python projects and need to keep the p
...
+
+``sh
+$ wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+$ sudo apt-get install apt-transport-https
+$ echo "deb https://artifacts.elastic.co/packages/5.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-5.x.list
+$ sudo apt-get update && sudo apt-get install elasticsearch
+$ sudo /bin/systemctl daemon-reload
+$ sudo /bin/systemctl enable elasticsearch.service
+```
+
```
Comment (dchukhin) [inline DETAILED_INSTRUCTIONS.md]: The Part 2 line ends up being part of a code block, rather than a heading

File: DETAILED_INSTRUCTIONS.md (hunk 1 above)
Comment (mlavin) [inline DETAILED_INSTRUCTIONS.md]: I'll fix though Meme-cached sounds more fun honestly.

Comment (daaray) [inline DETAILED_INSTRUCTIONS.md]: `Meme-cached` sounds like an amazing Ship It Day project.

**Pull request merged**